lib/plugin-api/hydrogen-kernel.js

```diff
@@ -1,36 +1,43 @@
-###
+/*
 * The `HydrogenKernel` class wraps Hydrogen's internal representation of kernels
 * and exposes a small set of methods that should be usable by plugins.
 * @class HydrogenKernel
-###
+*/
 
-module.exports =
-class HydrogenKernel
-    constructor: (@_kernel) ->
-        @destroyed = false
+export default class HydrogenKernel {
+    constructor(_kernel) {
+        this._kernel = _kernel;
+        this.destroyed = false;
```

Comment:
Note that `destroyed` is intentionally a part of the plugin API (i.e. plugins are expected and encouraged to use it). It can be switched to a property that proxies to `_kernel`, but removing it entirely is a breaking change. For a plugin API, stability is more important than perfection -- i.e. I would encourage not getting into the habit of making breaking changes.